var/spack/repos/builtin/packages/cuda/package.py

```diff
@@ -85,6 +86,14 @@ def setup_run_environment(self, env):
         env.set('CUDA_HOME', self.prefix)
 
     def install(self, spec, prefix):
+        if os.path.exists('/tmp/cuda-installer.log'):
+            try:
+                os.remove('/tmp/cuda-installer.log')
+            except Exception:
+                if spec.satisfies('@10.1:'):
```

Comment:
Why the version requirement? Is this a new feature?

Reply:
The crash seems to occur only occurs in these versions.